backend/capellacollab/settings/configuration/unifiedconfig/util.py

```diff
@@ -0,0 +1,47 @@
+# SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import capellacollab
+from capellacollab.config import config
+from capellacollab.settings.configuration import models as config_models
+from capellacollab.settings.configuration.unifiedconfig import models
+
+
+def get_metadata(
+    global_config: config_models.GlobalConfiguration,
+) -> models.Metadata:
+    return models.Metadata.model_validate(
+        global_config.metadata.model_dump()
+        | {
+            "version": capellacollab.__version__,
+            "host": config.general.host,
+            "port": str(config.general.port),
+            "protocol": config.general.scheme,
+        }
+    )
+
+
+def get_feedback(
+    global_config: config_models.GlobalConfiguration,
+) -> config_models.FeedbackConfiguration:
+    feedback = global_config.feedback
+    if not (config.smtp and config.smtp.enabled):
+        feedback.enabled = False
+        feedback.after_session = False
+        feedback.on_footer = False
+        feedback.on_session_card = False
+        feedback.interval.enabled = False
+
+    return feedback
+
+
+def get_navbar(
+    global_config: config_models.GlobalConfiguration,
+) -> config_models.NavbarConfiguration:
+    return global_config.navbar
+
+
+def get_beta(
+    global_config: config_models.GlobalConfiguration,
+) -> config_models.BetaConfiguration:
+    return global_config.beta
```

Comment:
```suggestion
```

Those two functions are overkill and just add another layer of abstraction. Write out `global_config.beta` directly where it's used.